docs/source/contributing.rst

```diff
@@ -3,79 +3,79 @@
 ============
 Contributing
```

Comment:
For this section, I made lots of edits. However, I was wondering if it shouldn't be redone to just point to the Contributing section in the PyAnsys Developer's Guide?

Reply:
This is a tough call. I feel that the contributing guide should be shared between repositories containing sections individual to each repository, thus minimizing code duplication.

Can you pull out what's specific to DPF and keep that here, and then just provide a link to the main contributing guide? I agree with the changes you made here, but I don't want you to have to make them in the other repositories as well.

Reply:
@akaszynski I agree with you as to first having a link to general contributing content and then having only PyDPF-specific contribution info. I'll do this in a subsequent PR if that is OK with you.

Reply:
That's perfect. Thanks @PipKat!